src/content/upgrades/merge/index.md

```diff
@@ -5,64 +5,236 @@ lang: en
 
-It is important to note that an implementation goal of The Merge is simplicity in order to expedite the transition from proof-of-work to proof-of-stake. Developers are focusing their efforts on this transition, and minimizing additional features that could delay this goal.
+Many dapps require a number of PoW block confirmations that take a period of time on par with how long PoS finality takes. Finality can offer additional security guarantees, but will not significantly speed up transactions.
+</ExpandableCard>
 
-**This means a few features, such as the ability to withdraw staked ETH, will have to wait a little longer after The Merge is complete.** Plans include a post-merge "cleanup" upgrade to address these features, which is expected to happen very soon after The Merge is completed.
+<ExpandableCard
+title="Misconception: &quot;You can withdraw staked ETH once The Merge occurs.&quot;"
+contentPreview="False. Staking withdrawals are not yet enabled with The Merge. These will have to wait until the Shanghai upgrade to follow.">
+Staked ETH, staking rewards to date, and newly issued ETH immediately after The Merge will still be locked on the Beacon Chain without the ability to withdraw.
+
+Withdrawal functionality is planned for the **Shanghai upgrade** to follow The Merge. This means that newly issued ETH, though accumulating on the Beacon Chain, will remain locked and illiquid for at least 6-12 months following The Merge.
+</ExpandableCard>
+
+<ExpandableCard
+title="Misconception: &quot;Validators will not receive any liquid ETH rewards til the Shanghai upgrade when withdrawals are enabled.&quot;"
+contentPreview="False. Fee tips/MEV will be credited to a Mainnet account controlled by the validator, available immediately.">
+This may seem counterintuitive to the above note that withdrawals are not enabled til the Shanghai upgrade, but validators WILL have immediate access to the fee rewards/MEV earned during block proposals.
+
+ETH is issued by the protocol as a reward to validators for contributing to consensus. This _newly issued_ ETH is accounted for on the Beacon Chain (consensus layer), where a validator has a unique address that holds its staked ETH and protocol rewards. This ETH is locked until Shanghai.
+
+ETH on the _execution layer_ (Mainnet as we know it) is accounted for separately from the consensus layer. When users execute transactions that involve Mainnet, ETH must be paid to cover the gas which includes a tip to the validator. This ETH is already on the execution layer, is NOT being newly issued by the protocol, and is available to the validator immediately (given a proper `fee recipient` address is provided to the client software).
+</ExpandableCard>
+
+<ExpandableCard
+title="Misconception: &quot;When withdrawals are enabled, stakers will all exit at once.&quot;"
+contentPreview="False. Validator exits are rate limited for security reasons.">
```

Comment:
Rate-limiting is developer terminology. We should try to speak in common language where possible... 

A more accessible way of phrasing this might be: 'The number of validators that can leave at once is capped for security reasons'


Given how buried this is I'm not concerned about going through this FAQ entirely and fixing it for now but something to consider if we talk about this anywhere else.